mlir/test/mlir-miopen-driver/e2e/conv2d_harness_cpu_verifier_kyxc_nhwc_nhwk_f16.mlir

```diff
@@ -1,8 +1,8 @@
-// RUN: mlir-miopen-driver -p -t f16 -fil_layout=gkyxc -in_layout=nhwgc -out_layout=nhwgk --host %s -c | mlir-rocm-runner --shared-libs=%rocm_wrapper_library_dir/librocm-runtime-wrappers%shlibext,%linalg_test_lib_dir/libmlir_runner_utils%shlibext --entry-point-result=void | FileCheck %s --check-prefix=E2E
+// RUN: mlir-miopen-driver -p -rand 1 -t f16 -fil_layout=gkyxc -in_layout=nhwgc -out_layout=nhwgk --host %s -c | mlir-rocm-runner --shared-libs=%rocm_wrapper_library_dir/librocm-runtime-wrappers%shlibext,%linalg_test_lib_dir/libmlir_runner_utils%shlibext --entry-point-result=void | FileCheck %s --check-prefix=E2E
```

Comment:
please create new cases instead of modifying existing cases.

Reply:
This is actually not a new test case. It intends to do the same thing as the old test. As I've changed mlir-miopen-driver.cpp, '-rand 1' is needed for -pv to behave the same as the old test without '-rand 1'.

Reply:
I see.